src/frontend/screens/Settings/ProjectConfig.tsx

```diff
@@ -114,26 +117,85 @@ const ProjectConfig = () => {
         <Text style={[styles.centerText, styles.projectKey]}>
           {projectKeySlice ? projectKeySlice + "**********" : "MAPEO"}
         </Text>
+
+        {!!configName && (
+          <React.Fragment>
+            <Text style={[styles.centerText, { marginTop: 10 }]}>
+              <FormattedMessage {...m.name} />
+            </Text>
+            <Text style={[styles.centerText, styles.projectKey]}>
+              {configName || ""}
```

Comment:
the fallback shouldn't be necessary here because of the `!!configName` check

Reply:
Right, I originally did this, and then decided to use !!configName afterwards and forgot to take it out. good catch